### ISSUE TITLE
Some requested features inspired by My Life Organized

### ISSUE THREAD

**Issue by dnrce** [TracksApp/tracks#564]
_Migrated from the original issue at https://www.assembla.com/spaces/tracks-tickets/tickets/564_

BSAG

Hey, I am checking out tracks, and liking it so far, despite the hassle of getting to run on windows. I am currently using My Life Organized http://www.mylifeorganized.net/ as my GTD app. It has 3 panes, your Outline, your To-DO and your places (contexts with many features). 
Now, I like it's outlining features, as it allows you to organize and compartmentalize your planning and task tracking. For instance, Right now I 4 major areas of my Outline, a motivation, a Planning, Execution, and a "Reference". WIthin the planning and and execution I have it divided into Roles, Professional & creative and and Personal and home. Within those I various headers, inbox, quicklist, standalone events, single step actions, punchlists & routines, etc. And way down there I have my Traditional GTD contexts as @calls, @computer, @errands, etc. 

You can also zoom in on a particular branch or area, say you only want to see/work with Professional & Creative tasks.

I am attemping to do this via Tracks contexts, and it is leading to a rather long lists of contexts. Which brings me to request. MLO uses what it terms Places, which are essentially contexts. However they have two sets of handy properties. First, you can assign hours to each place. You can assign times when this context is open, and will show up in the ToDo list, and when it is closed. This is a very handy feature for setting working time, free time, home time, etc. The todo list also has a toggle to show closed places.  

The other big feature is that you have Places that include other places. So you can setup daily routines that are included in a larger Routines & appointments category, You can have your todo list include various segments of work time. For instance, I have a weekly work time place that is made up of my morning workday, afternoon wrkday , evening workday. As well as for weekends. I also have my free time and home time categories. 

All of these are included in either my ToDo list or Routines & appointments list. Thus, when I look at the To-Do pane under the relevant master place/context, It will show me only the tasks relevant to what I am doing. Or I can focus on specific contexts, such as the traditional @calls etc. (You can also toggle it show closed places, which often leads to information overload.)

So, in short, if you could implement these two features. First a feature allowing contexts to include other contexts. Second, a feature allowing you to assign open & closed hours to a context, that'd make it much more useful as a planning/capturing tool. Especially if it would show the included contexts in a collapsible menu, and would allow you to focus on a particular branch of your outline/context list. 

thanks

Rob

_Originally reported by Anonymous on September 9, 2007 at 02:06:06 (+0300)_
